tests/addresslib/parser_mailbox_test.py

```diff
@@ -540,10 +531,6 @@ def test_local_part():
         EmailAddress(sample_qtext, '"{0}"@b'.format(sample_qtext)))
 
     # fail qtext
-    run_mailbox_test('"{0}""{0}"@b'.format(sample_qtext), None)
```

Comment:
Valid address because we allow `Foo foo@bar.com` without `<>`. Equivalent to `"{}" {}@b`.